### PR TITLE
prometheus: add documentation

### PR DESCRIPTION
**Pull request by adelcast**
# Description
Add documentation page for Prometheus integration

# Checklist
- [x] The PR is tagged with proper labels (bug, enhancement, feature, documentation)
- [ ] I have performed a self-review of my own code
- [ ] I have added unit tests that prove my fix is effective or that my feature works
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] If new documentation has been added, relative paths have been added to the appropriate section of `docs/mint.json`

cc:
@dy46

### CODE REVIEW

File: docs/integrations/observability/prometheus.mdx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+enable Promethues style metrics on the <BASE_PATH>/metrics route.
+
+## Prometheus metrics
+
+When enabled, Mage will publish Metrics for the HTTP Server (Tornado) as well as for
+the Python Runtime. The /metrics route will be updated on-demand to respond to scrape requests.
+
+
+
+
```
Comment (dy46) [inline docs/integrations/observability/prometheus.mdx:18]: can we remove these extra line

File: docs/integrations/observability/prometheus.mdx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+title: "Monitoring with Prometheus"
+sidebarTitle: "Prometheus"
+---
+
+## Enable Prometheus
+
+You will need to add `ENABLE_PROMETHEUS` environment variable as `true` in order to
+enable Promethues style metrics on the <BASE_PATH>/metrics route.
```
Comment (dy46) [inline docs/integrations/observability/prometheus.mdx:9]: typo on "Promethues"

Comment (adelcast) [reply docs/integrations/observability/prometheus.mdx]: oopps....done

File: docs/integrations/observability/prometheus.mdx (hunk 1 above)
Comment (adelcast) [reply docs/integrations/observability/prometheus.mdx]: fixed

**Review by dy46 — APPROVED** [review]

**Pull request merged**